galaxy.yml

```diff
@@ -0,0 +1,35 @@
+namespace: 'hifis'
+name: 'software_services'
+version: '4.0.0'
+readme: 'README.md'
+authors:
+- 'HIFIS Software Services <support@hifis.net>'
+description: 'This collection provides production-ready Ansible roles used for providing services in the field of research software.'
+license:
+  - 'GPL-2.0-or-later'
+license_file: ''
```

Comment:
This key is mutually exclusive with `license`. It may happen that we have to list several licenses here. I would address the proper licensing of the collection and the roles that will be added, some of which have different licenses, in a separate issue (#197). Therefore, I'll drop the key.